lib/queue.rb

```diff
@@ -1,28 +1,61 @@
+# Wave 2
+# require_relative './stack.rb'
+require "./linked_list"
```

Comment:
This should be a require_relative, although you don't need a linked list in this class for the assignment.